uniffi/src/lib.rs

```diff
@@ -2,19 +2,25 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#[cfg(feature = "clap")]
-mod cli;
 /// Reexport items from other uniffi creates
+pub use uniffi_core::*;
+pub use uniffi_macros::{export, include_scaffolding, Enum, Error, Object, Record};
```

Comment:
This could also be behind a feature gate, but I skipped that since it's a proc-macro crate.  I'm not really sure how this works, but I believe proc-macro crates don't contribute to the final binary size.